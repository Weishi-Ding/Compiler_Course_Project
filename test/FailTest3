interface myInterface {
    int sum (int a, int b);
}

/*class must implement methods in interface*/
class myClass implements myInterface {
    int multiply(int a, int b) {
        return a * b;
    }
    
    constructor myClass() {
        
    }
}

int main () {
    
}