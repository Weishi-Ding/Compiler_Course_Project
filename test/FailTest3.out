Fatal error: exception Dune__exe__Semant.Exception("sum of interface myInterface is not defined in class myClass")
